llvm/include/llvm/MC/MCInstrDesc.h

```diff
@@ -105,6 +107,16 @@ class MCOperandInfo {
   /// Set if this operand is a branch target.
   bool isBranchTarget() const { return Flags & (1 << MCOI::BranchTarget); }
 
+  /// Set if this operand is a branch target.
+  bool mustBeUnsealedCapability() const {
+    return Flags & (1 << MCOI::MustBeUnsealedCapability);
+  }
+
+  /// Set if this operand is a branch target.
+  bool mustBeTaggedCapability() const {
+    return Flags & (1 << MCOI::MustBeUnsealedCapability);
```

Comment:
```suggestion
    return Flags & (1 << MCOI::MustBeTaggedCapability);
```

Doesn't matter right now since we don't do useful checks, but clearly needs to be fixed.